### ISSUE TITLE
Workbench: Validation error not reported

### ISSUE THREAD

**Issue by PetrToman** [encog/encog-java-core#62]
1. Generate .ega file using Analyst Wizard in Workbench using data from http://dione.zcu.cz/~toman40/encog/numbers.zip (10 classes of data representing 20x20 images of handwritten digits "0"-"9").
2. Run Analyst Wizard tasks, generate `numbers_eval.csv` and click "Stop" when the training starts (needed in order to perform next steps).
3. Generate `numbers_eval.egb` by "Normalize To Training (EGB)" action.
4. Doubleclick .eg file (Basic network), click "Train", select `numbers_train.egb` as Training Set and `numbers_eval.egb` as Validation Set. Then choose RPROP with default values.
5. Start training -> instead of "Validation Error" value there will be a square sign.
